src/content/en/updates/posts/2016/03/device-mode-v2.markdown

```diff
@@ -0,0 +1,112 @@
+---
+layout: updates/post
+title: "A new Device Mode for a mobile-first generation"
+description: "Learn all about Chrome DevTools' new, refreshed Device Mode introduced in Chrome 49."
+published_on: 2016-03-16
+updated_on: 2016-03-16
+authors:
+  - pbakaus
+tags:
+  - devtools
+  - update
+  - chrome49
+featured_image: /web/updates/images/2016/03/device-mode-v2/device-mode-initial-view.png
+---
+
+# A new Device Mode for a mobile-first generation
+
+We've introduced the Device Mode, a way to emulate devices and work with your 
+responsive design, a bit more than a year ago, and it's now time for its first 
+major upgrade, starting in Chrome 49. So what's new?
+
+While we offered ways to emulate mobile in the past, the development default has 
```

Comment:
**Mobile is becoming the starting point in Chrome DevTools**. While we offered ways to emulate mobile in the past, the development default was desktop. Mobile emulation always had to be turned on. Now that consumption of mobile sites has overtaken desktop in many places, we're switching our position in DevTools as well.